### DISCUSSION TITLE
希望增加“瀑布流”组件

### DISCUSSION THREAD

**Discussion by THUNDZENG** [Moonofweisheng/wot-design-uni#154] [Ideas]
有电商场景小程序，商品列表需要用瀑布流展示，所以提了这个建议

**Comment by Moonofweisheng** [comment]
瀑布流确实是比较常见的场景，会加入开发计划的

**Comment by Moonofweisheng** [comment]
见[2024开发计划](https://github.com/users/Moonofweisheng/projects/1)

**Comment by elarbor** [comment]
这个组件还真是比较常用

**Comment by killerddd3** [comment]
#1424 想问下为什么回退了，这个组件有计划吗? @Moonofweisheng

**Comment by HuayangZhao** [comment]
我看25年8月份新增了瀑布流组件，但新版wot-ui中直接无了，后面还会有嘛
